server/cpho/scripts/dev.py

```diff
@@ -26,7 +26,7 @@
 @transaction.atomic
 def run():
     create_admins()
-    create_data()
+    # create_data()
```

Comment:
is this supposed to go into main like this?

Reply:
good catch!
I had it commented out as I wanted to seed my local db with data from prod.
its better to revert this (even though we're not going to seed prod anyway)